ITEvents/Screens/Search tab/SearchPresenter.swift

```diff
@@ -135,8 +135,8 @@ class SearchPresenter: ISearchPresenter {
         self.events.append(contentsOf: fetchedEvents)
         self.eventViewModels.append(contentsOf: fetchedEvents.map(createEventViewModel))
         
-        self.view.setEvents(self.eventViewModels)
         self.view.hideLoadingIndicator()
+        self.view.setEvents(self.eventViewModels)
```

Comment:
why we are changing the order of the lines?

Reply:
Because when hideLoadingIndicator called, it change isLoadingIndicatorShown property. And this property needed for needToShowBackground property. So if we don't change the order, message will not be shown.